src/client_eosc.py

```diff
@@ -1,27 +1,25 @@
-import json
```

Comment:
Do we actually need this file? I think it can be fully replaced with "oms-adapter-jira".